src/popup/js/templates.js

```diff
@@ -294,6 +294,32 @@ const getPopupPaperIconsHTML = (paper, currentUrl, is) => {
         </div>`;
     }
 
+    let vanity = "";
```

Comment:
Why not add SciRate, Vanity and Ar5iv to the global view (not popup): line 100 (getMemoryItemHTML)

Reply:
Not especially, let's keep it simple for now and maybe figure out a better way to integrate if this is wanted by others than just me ;)